core/src/main/java/io/questdb/cairo/security/ReadOnlySecurityContext.java

```diff
@@ -180,6 +180,21 @@ public void authorizeInsert(TableToken tableToken, @NotNull ObjList<CharSequence
         throw CairoException.authorization().put("Write permission denied").setCacheable(true);
     }
 
+    @Override
+    public void authorizeLineAlterTableAddColumn(TableToken tableToken) {
```

Comment:
nit: "Line" is a little ambiguous, may be ILP? We refer to this context as LineTCP, UDP is unused, may be we can do LineTCP here? Wdyt?

Reply:
I considered `Ilp`, but we seem to use `Line` instead of it consistently in the code base. As for `LineTcp`, we may want to support auth for UDP one day and then we'd have to rename these methods, so taking that into account I'd prefer `Line`.